airflow/models.py

```diff
@@ -3055,6 +3055,34 @@ def latest_execution_date(self):
         session.close()
         return execution_date
 
+    def descendants(self, dagbag, task_ids=None, include_downstream=False,
```

Comment:
No, because it is outside of the dag parser. We test if for Dags but not between Dags. Your code could get into an endless loop (Dag A -> Dag B -> Dag A -> Dag B etc.)